docs/connectors/slack.md

```diff
@@ -87,6 +88,27 @@ class GreeterSkill(Skill):
         await message.respond("Hi")
 ```
 
+## Get messages from History
+Sometimes you need to search trough the history of a channel. For this you can use the `search_history_messages` method from the slack connector which returns all the messages on a specified range of time.
+
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_regex
+
+class SearchMessagesSkill(Skill):
+    """This is the most simple form of a skill"""
+
+    @match_regex(r"search messages")
+    async def search_messages(self, message):
+        """"""
+        slack = self.opsdroid.get_connector("slack")
+        messages = await slack.search_history_messages(
+            "CHANEL_ID", start_time="1512085950.000216", end_time="1512104434.000490"
+        )
+        await message.respond(messages)
+```
```

Comment:
Could you move this into the docstring of the `search_history_messages` method and then use autofunc here to include it in the docs.

Helps avoid docs getting out of sync with code.

Reply:
Moved into the docstring